sourcetracker/sourcetracker.py

```diff
@@ -843,3 +855,106 @@ def _cli_loo_runner(sample, source_category, alpha1, alpha2, beta, restarts,
     o = open(os.path.join(output_dir, sample + '.txt'), 'w')
     o.writelines(lines)
     o.close()
+
+
+def _api_gibbs(source_df, sink_df, alpha1, alpha2, beta, restarts,
```

Comment:
might go with an easier name that's less fussy to type. I think just `gibbs` would do here.